14_system-prep-for-build.Rmd

```diff
@@ -1,9 +1,66 @@
 # Setup an R dev environment
 
-System prep necessary to install R packages from source.
+There are two different types of R packages you can install, **binary**
+packages and **source** packages. When you install released packages from CRAN
+using `install.packages()` you are installing **binary** packages. When you
+install development packages using `devtools::install_github()` you are
+installing **source** packages.
 
-In fact, what is a source package vs a binary?
+We recommend the `devtools` package to install source packages from GitHub and
+elsewhere, and also to install (and develop) packages you may write yourself.
 
-<http://stat545.com/packages01_system-prep.html>
+`devtools` can be installed from CRAN like any other package.
 
-Certain sections and graphics from here: http://r-pkgs.had.co.nz/package.html
+```r
+install.packages("devtools")
+library(devtools)
+```
+
+Once a **binary** or **source** package is installed they are identical,
+however **source** packages which include C/C++ or Fortran code require
+additional tools to install, so there is additional system preparation needed.
+
+## Windows: system prep
+
+On windows the compiler collection needed for installing packages with C/C++ or
+Fortran code in them is called `Rtools`.
```

Comment:
I think of Rtools as also bringing other unix-y things, such as tar. I propose more general wording:

> On Windows, the collection of tools needed to install packages from source is called Rtools.